packages/auth/src/features/system/handler.ts

```diff
@@ -114,5 +114,6 @@ export async function registerSystemClient(
   return res
 }
 export const registerRquestSchema = Joi.object({
-  scope: Joi.string()
+  type: Joi.string(),
+  name: Joi.string()
```

Comment:
I think we should be able to deprecate this endpoint now that we have graphql mutation. @euanmillar What do you think?